src/modules/users/components/user-create.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { connect } from "react-redux";
+
+import { UserCreateForm } from "modules/users/components/user-form";
+import { UserActionCreators } from "modules/users/user-actions";
+
+const DumbUserCreate = ({ saveUser }) => <UserCreateForm onSubmit={saveUser} />;
+
+const mapStateToProps = state => ({});
+
+const mapDispatchToProps = {
+  saveUser: UserActionCreators.saveUser
+};
+
+export const UserCreate = connect(
+  mapStateToProps,
```

Comment:
You can use `null` or `undefined` if you don't need `mapStateToProps`.

Reply:
Cheers, I didn't know that - updated.